zwave_js_server/client.py

```diff
@@ -176,12 +188,23 @@ async def _handle_connection(self):
             )
             self.tries = 0
 
-            self._logger.info("Connected")
+            version_msg = await client.receive_json()
+
+            self._logger.info(
+                "Connected to Home %s (Server %s, Driver %s)",
+                version_msg["homeId"],
+                version_msg["serverVersion"],
+                version_msg["driverVersion"],
+            )
             self.state = STATE_CONNECTED
 
             if self._on_connect:
                 await gather_callbacks(self._logger, "on_connect", self._on_connect)
 
+            await client.send_json(
+                {"messageId": uuid.uuid4().hex, "command": "start_listening"}
```

Comment:
you might want to move creation of default id to the send_json method perhaps so it can be omitted ?

Reply:
but that means the send json needs to check for messageId key

Reply:
Will do that in the next pass when adding full support for commands.